modules/python-modules/syslogng/modules/hypr/__init__.py

```diff
@@ -123,22 +124,21 @@ def init(self, options):
         # Set max_performance if defined
         self.max_performance = False
         if "max_performance" in options:
-            if options["max_performance"].lower() == "true":
-                self.max_performance = True
+                self.max_performance = options["max_performance"]
 
         # Default time to go back is 4 hours
         initial_hours = 4
         if "initial_hours" in options:
 
             # If r is set as an initial_hours value, ignore persistence
-            if "r" in options["initial_hours"]:
-                self.logger.info("Disabling persistence due to special initial_hours setting (%s)", options["initial_hours"])
+            if "r" in str(options["initial_hours"]):
```

Comment:
We use the str() on initial_hours. Shouldn't we leave the quotes on the SCL side?

Reply:
At the final version `initial_hours` is not used as an int, and not casted to str and parsed, etc.